packages/devtools_app/lib/src/timeline/timeline_flame_chart.dart

```diff
@@ -995,8 +1006,8 @@ class SelectedFrameBracketPainter extends FlameChartPainter {
     this.selectedFrame, {
     @required double zoom,
     @required BoxConstraints constraints,
-    @required double verticalScrollOffset,
-    @required double horizontalScrollOffset,
+    @required ScrollController verticalScroll,
+    @required LinkedScrollControllerGroup horizontalScroll,
```

Comment:
nit: naming. to keep this consistent with other parts of the code, can we add controller and controllerGroup to the names, respectively